src/ui/issuecolumn/ColumnControl.java

```diff
@@ -229,8 +229,19 @@ public void setCurrentlyDraggedColumnIndex(int i) {
 	
 	public void closeCurrentColumn() {
 		if (currentlySelectedColumn.isPresent()) {
-			closeColumn(currentlySelectedColumn.get());
-			currentlySelectedColumn = Optional.empty();
+			int columnIndex = currentlySelectedColumn.get();
+			System.out.println("Going to close column " + columnIndex);
```

Comment:
We should use a logger instead. `System.out.println` has no guarantees of thread safety (and so messages may appear out of order in concurrent scenarios), and won't write to log files.